app/views/exam_templates/_create_exam_template_info.html.erb

```diff
@@ -0,0 +1,12 @@
+<div id="create-template-info-modal-content">
+  <div class="inline-labels">
+    <%= f.label :name, ExamTemplate.human_attribute_name('name'), class: "required" %>
+    <%= f.text_field :name, required: true %>
+    <%= f.label :file_io, t('exam_templates.create.upload'), class: "required" %>
+    <%= f.file_field :file_io, required: true %>
+  </div>
+  <section class='dialog-actions'>
+    <input id="create-template-info-next" class='create-template-view-change' type='button' value='Next'/>
```

Comment:
After playing around with this, I don't think we need the template divisions in the modal. (I didn't realize this until using it myself, and seeing that there's not much of a delay between saving the exam template and then seeing the "edit" pane appear, where we can configure template divisions and other settings).

Let's stick with the modal just having the exam template file, and then after the user saves that, they do other configuration in the pane, not in the modal. This will also simplify the code by removing a bunch of the Javascript/HTML you added (sorry I want you to now remove it).